client/auth/login.jsx

```diff
@@ -67,16 +67,36 @@ module.exports = React.createClass( {
 
 	componentWillUnmount: function() {
 		AuthStore.off( 'change', this.refreshData );
+		clearTimeout( this.pollTimeout );
+		this.pollTimeout = null;
 	},
 
 	refreshData: function() {
 		this.setState( AuthStore.get() );
 	},
 
+	verifyPushAuthentication: function( pushauth ) {
+		const POLL_INTERVAL = 6000;
```

Comment:
how about moving this into the module area above the class to `React.createClass()`? It doesn't really have a need to be defined inside the function, or at least not as a `CONSTANT` inside the function.